app/src/main/java/com/github/se/project/ui/components/LessonEditor.kt

```diff
@@ -237,6 +302,20 @@ fun LessonEditor(
                   style = MaterialTheme.typography.headlineMedium,
                   textAlign = TextAlign.Center)
 
+              if (canBeInstant.value) {
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    modifier =
+                        Modifier.testTag("instantColumn")
+                            .padding(vertical = 0.dp, horizontal = 0.dp)) {
+                      Text("Now", style = MaterialTheme.typography.labelSmall)
+                      Switch(
+                          checked = instant.value,
+                          onCheckedChange = { instant.value = !instant.value },
+                          modifier = Modifier.testTag("instantSwitch"))
+                    }
+              }
+
```

Comment:
<img width="394" alt="image" src="https://github.com/user-attachments/assets/c59bbd93-7200-485d-b0f4-854df8b9ce3a">
Find a way to align better the elements.